src/System.Console/ref/System.Console.cs

```diff
@@ -36,8 +36,11 @@ public static partial class Console
         public static void MoveBufferArea(int sourceLeft, int sourceTop, int sourceWidth, int sourceHeight, int targetLeft, int targetTop, char sourceChar, ConsoleColor sourceForeColor, ConsoleColor sourceBackColor) { }
         public static bool NumberLock { get { return default(bool); }}
```

Comment:
Yes that is correct for the version update but it doesn't include the package update. For now I think we are landing in a place where people porting APIs in the dev/api branch shouldn't worry about contract or package versioning and we will do that in one full pass before we merge back into master. It is fine for you to keep your version change however.